rpcserver.go

```diff
@@ -2317,6 +2317,13 @@ func (r *rpcServer) CloseChannel(in *lnrpc.CloseChannelRequest,
 				"is offline (try force closing it instead): %v", err)
 		}
 
+		// If conf-target is not set then use the conf-target used for
+		// co-op closes that are initiated by the remote peer.
+		targetConf := uint32(in.TargetConf)
+		if targetConf == 0 {
```

Comment:
This breaks existing behavior as `DetermineFeePerKw` only allows the fee rate or the conf target to be set. With this change, users can no longer set _just_ the fee rate.